less/less.d.ts

```diff
@@ -72,6 +77,8 @@ declare namespace Less {
 }
 
 interface LessStatic {
+    options: Less.StaticOptions;
```

Comment:
errr, just saw the [update to V2](http://lesscss.org/usage/#v2-upgrade-guide-browser-usage). seems good